qcify.py

```diff
@@ -0,0 +1,128 @@
+#!/data/bnf/sw/miniconda3/bin/python
+from pprint import pprint
```

Comment:
```suggestion
from collections import defaultdict
```

`defaultdict` is used for a suggested rewrite below

`pprint` is not actually used in the script